<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GYDManager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>5</number>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="upload">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #d1d1d6;
    border-radius: 8px;
	color: #000000;
	padding: 10px 20px 10px 20px;
	font-size: 20px;
}

QPushButton:pressed {
    background-color: #e6e6eb;
}

QPushButton:disabled {
    background-color: #f2f2f7;
    color: #8e8e93;
    border-color: #d1d1d6;
}</string>
            </property>
            <property name="text">
             <string>Upload</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/menu/static/upload.png</normaloff>:/menu/static/upload.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="left_menu">
          <property name="styleSheet">
           <string notr="true">QTreeWidget {
    background-color: #ffffff;
    border: 1px solid #d1d1d6;
    border-radius: 3px;
    color: #000000;
    selection-background-color: #007aff;
    selection-color: #ffffff;
}</string>
          </property>
          <property name="indentation">
           <number>15</number>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>1</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Tasks</string>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/menu/static/swap_vert.png</normaloff>:/menu/static/swap_vert.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Drive</string>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/menu/static/cloud.png</normaloff>:/menu/static/cloud.png</iconset>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QProgressBar" name="storage_status_progressbar">
              <property name="styleSheet">
               <string notr="true">QProgressBar {
	max-height: 5px;
}</string>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="textVisible">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="storage_status_label">
              <property name="text">
               <string>Used ? from ?</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="settings">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #d1d1d6;
    border-radius: 5px;
    color: #000000;
    padding: 2px 8px;
}

QPushButton:pressed {
    background-color: #e6e6eb;
}

QPushButton:disabled {
    background-color: #f2f2f7;
    color: #8e8e93;
    border-color: #d1d1d6;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/menu/static/settings.png</normaloff>:/menu/static/settings.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QStackedWidget" name="content_widget">
       <property name="minimumSize">
        <size>
         <width>700</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tasks_widget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tasks_table">
           <property name="styleSheet">
            <string notr="true">QTableWidget {
	background-color: #ffffff;
	border: 1px solid #d1d1d6;
	border-radius: 3px;
	color: #000000;
	selection-background-color: #007aff;
	selection-color: #ffffff;
}</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Size</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Speed</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Status</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Progress</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ETA</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="drive_widget">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>3</number>
           </property>
           <item>
            <widget class="QPushButton" name="back">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #d1d1d6;
    border-radius: 5px;
	color: #000000;
    padding: 3px 12px;
}

QPushButton:pressed {
    background-color: #e6e6eb;
}

QPushButton:disabled {
    background-color: #f2f2f7;
    color: #8e8e93;
    border-color: #d1d1d6;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/drive/static/arrow_back.png</normaloff>:/drive/static/arrow_back.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="forward">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #d1d1d6;
    border-radius: 5px;
	color: #000000;
    padding: 3px 12px;
}

QPushButton:pressed {
    background-color: #e6e6eb;
}

QPushButton:disabled {
    background-color: #f2f2f7;
    color: #8e8e93;
    border-color: #d1d1d6;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/drive/static/arrow_forward.png</normaloff>:/drive/static/arrow_forward.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="refresh">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #d1d1d6;
    border-radius: 5px;
	color: #000000;
    padding: 3px 12px;
}

QPushButton:pressed {
    background-color: #e6e6eb;
}

QPushButton:disabled {
    background-color: #f2f2f7;
    color: #8e8e93;
    border-color: #d1d1d6;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/drive/static/refresh.png</normaloff>:/drive/static/refresh.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="search_line">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #d1d1d6;
    border-radius: 3px;
    color: #000000;
	height: 24px;
}

QLineEdit:focus {
    border: 1px solid #007aff;
    background-color: #ffffff;
}

QLineEdit:disabled {
    background-color: #f2f2f7;
    color: #8e8e93;
    border-color: #d1d1d6;
}</string>
             </property>
             <property name="placeholderText">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="files_view">
           <property name="styleSheet">
            <string notr="true">QTableView {
	background-color: #ffffff;
	border: 1px solid #d1d1d6;
	border-radius: 3px;
	color: #000000;
	selection-background-color: #007aff;
	selection-color: #ffffff;
}</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>150</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar {
    border-top: 1px solid #d1d1d6;
}</string>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionUpload"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_folder"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="actionTrash"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSort"/>
    <addaction name="separator"/>
    <addaction name="actionrefresh"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuview"/>
   <addaction name="menuhelp"/>
  </widget>
  <action name="actionCreate_folder">
   <property name="text">
    <string>Create folder</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="text">
    <string>Upload</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionTrash">
   <property name="text">
    <string>Trash</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSort">
   <property name="text">
    <string>Sort</string>
   </property>
  </action>
  <action name="actionrefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
